OWNERS

```diff
@@ -9,15 +9,14 @@ approvers:
   - mheon
   - mtrmac
   - rhatdan
```

Comment:
Not listed as a maintainer in this PR's [MAINTAINERS.md](https://github.com/containers/podman/pull/25398/files#diff-39da3bd6270d44ea37b6ed50bd42eeb9d93ac5e1639645871a69cbe08cbe29deR29).

Reply:
Good catch